partials/eligibility.html

```diff
@@ -0,0 +1,102 @@
+          ${{earningRecord.taxedEarnings | number:0}}</td>
+        <td headers="epc">${{recipient.getEarningsPerCreditForYear(earningRecord.year) | number:0}}</td>
+        <td headers="credits" ng-show="earningRecord.credits >= 0">
+          {{earningRecord.credits | number:0}}</td>
+      </tr>
+    </tbody>
+    </table>
+  </span>
+
+  <p>
+    <span ng-if="futureYears.length > 0">If you work {{recipient.futureEarningsYears | number:0}} 
+      additional years, you will earn {{recipient.plannedCredits | number:0}} additional credits. 
+    </span>
+  </p>
+
+  <span ng-if="futureYears.length > 0">
+    <table class="fancy-table">
+    <thead>
+      <tr>
+        <th id="workyear">Year</th>
+        <th id="age">Age</th>
+        <th id="taxedearnings">Taxed Earnings</th>
+        <th id="epc">Earnings Per Credit</th>
+        <th id="credits">Credits<th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr ng-repeat="year in futureYears">
+        <td headers="workyear">{{year}}</td>
+        <td headers="age">{{recipient.exampleAge(year).age}}</td>
+        <td headers="taxedearnings">${{futureWageWorkSlider.minValue | number:0}}</td>
+        <td headers="epc">${{recipient.getEarningsPerCreditForYear(2018) | number:0}}</td>
+        <td headers="credits">{{recipient.calculateCredits(futureWageWorkSlider.minValue, 2018) | number:0}}</td>
+      </tr>
+    </tbody>
+    </table>
+  </span>
+  
+  <div ng-if="recipient.hasEarningsBefore1978" class=insetTextBox>
+    <h4>Special Rule</h4>
+    <p>
+      Earnings before 1978 were reported quarterly, and you earned one credit per quarter.
+      Because this website uses yearly earnings, credits may not be calculated correctly for
+      years before 1978.
+    </p>
+  </div>
+
+  <p>
+    <span ng-if="recipient.earnedCredits >= 40">You have earned the required <b>40</b> credits to be 
+      <u>permanently eligible</u> for retirement benefits on your own work record. 
+    </span>
+    <span ng-if="40 > recipient.earnedCredits && recipient.totalCredits >= 40">You will have <b>40</b> credits
+      when you complete <b>{{recipient.neededYears | number:0}}</b> additional years of work earning 
+      <b>${{futureWageWorkSlider.minValue | number:0}}</b> per year.  You will become <u>permanently eligible</u>
+      for retirement benefits on your own work record when you have earned 40 credits.
+    </span>
+    <span ng-if="40 > recipient.totalCredits">You have earned <b>{{recipient.earnedCredits | number:0}}</b> credits 
+      and will earn <b>{{recipient.plannedCredits | number:0}}</b> credits for a total of 
+      <b>{{recipient.totalCredits | number:0}}</b> credits, which is not enough to be eligible for retirement benefits 
+      on your own work record.  You may be eligible for spousal benefits.  You will become 
+      <u>permanently eligible</u> for retirement benefits on your own work record when you have earned 
+      40 credits.
+    </span>
+  </p>
+
+  <div class=insetTextBox>
```

Comment:
I'd hide this section if `recipient.earnedCredits >= 40`, since it may be slightly confusing if the person already has sufficient credits.